### PR TITLE
Unify SMPL-like models to mesh models

### PR DESCRIPTION
**Pull request by zengwang430521**

### CODE REVIEW

**Review by jin-s13 — APPROVED** [review]

File: mmpose/models/__init__.py [hunk 1]
```diff
@@ -1,13 +1,15 @@
 from .backbones import *  # noqa
-from .builder import (BACKBONES, HEADS, LOSSES, NECKS, POSENETS,
-                      build_backbone, build_head, build_loss, build_neck,
-                      build_posenet)
+from .builder import (BACKBONES, EXTERNAL_MODELS, HEADS, LOSSES, NECKS,
```
Comment (innerlee) [inline mmpose/models/__init__.py:2]: What does `EXTERNAL_MODELS` mean?

**Comment by innerlee** [comment]
What does "compact" mean?

File: mmpose/models/__init__.py (hunk 1 above)
Comment (jin-s13) [reply mmpose/models/__init__.py:2]: Currently, it only contains SMPL. Any suggestions on the name?

Comment (innerlee) [reply mmpose/models/__init__.py:2]: Something like `MESH_LAYERS`?

Comment (ly015) [reply mmpose/models/__init__.py:2]: I agree with the name `MESH_LAYERS`.
And I suggest moving the layers as well as the registry and builder into `mmpose/models/utils`.

**Review by innerlee — APPROVED** [review]

**Review by ly015 — APPROVED** [review]

**Pull request merged**